src/pages/api/identity/get-identity.ts

```diff
@@ -56,13 +58,20 @@ const handler: Handler = async (request, response) => {
     }).catch((error) => {
       span.setDisclosedAttribute("is Identity fetched", false);
 
-      if (error instanceof GraphqlErrors) {
+      if (error instanceof IdentityNotFoundError) {
         throw webErrorFactory({
           ...webErrors.identityNotFound,
           parentError: error,
```

Comment:
I think we don't need to pass the `parentError` in this case, since it's a waited and known exception, the concatened messages of both errors will be duplicates in the end. What do you think ?

Reply:
That's actually a good point. It will make the error message way cleaner 👍 .